util.js

```diff
@@ -34,10 +34,10 @@ module.exports.nonZeroOrNA = nonZeroOrNA
  * Computes message for Jade template from data returned from memcache.
  */
 module.exports.messageFromCacheData = function(v) {
+  const d = (typeof v === 'string') ? JSON.parse(v) : v
```

Comment:
I think you can get rid of this check? Seems like v is always a parsed JSON object at this point.